pkg/services/ngalert/schedule/registry.go

```diff
@@ -106,20 +111,15 @@ func (a *alertRuleInfo) eval(eval *evaluation) (bool, *evaluation) {
 // update sends an instruction to the rule evaluation routine to update the scheduled rule to the specified version. The specified version must be later than the current version, otherwise no update will happen.
 func (a *alertRuleInfo) update(lastVersion ruleVersionAndPauseStatus) bool {
 	// check if the channel is not empty.
-	msg := lastVersion
 	select {
-	case v := <-a.updateCh:
-		// if it has a version pick the greatest one.
-		if v.Version > msg.Version {
-			msg = v
-		}
+	case <-a.updateCh:
```

Comment:
This is not needed since after https://github.com/grafana/grafana/pull/64780 only scheduler sends the commands